distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/JvmOptionsParser.java

```diff
@@ -133,12 +131,19 @@ private List<String> jvmOptions(final Path config, Path plugins, final String es
         final List<String> ergonomicJvmOptions = JvmErgonomics.choose(substitutedJvmOptions);
         final List<String> systemJvmOptions = SystemJvmOptions.systemJvmOptions();
 
+        final List<String> apmOptions = APMJvmOptions.apmJvmOptions(
+            args.nodeSettings(),
+            args.keystore(),
+            Path.of(substitutions.get("ES_TMPDIR"))
```

Comment:
We have the tmpDir outside of the call to jvmOptions. Let's pass it through instead of assuming the env var name?